apps/issuer/models.py

```diff
@@ -507,7 +512,48 @@ def notify_admins(self, badgr_app=None, renotify=False):
         adapter = get_adapter()
         for user in users:
             adapter.send_mail(template_name, user.email, context=email_context)
+ 
+    # Notify Issuer owner when issuer gets verified
+    def notify_issuer_owner(self, badgr_app=None, renotify=False):
+        """
+        Sends an email notification to the Issuer owner.
+        """
+
+        try:
```

Comment:
A `try catch` seems rather hacky here, isn't it possible to check if the `EmailBlacklist.objects` collection contains the mail directly?

Reply:
It doesn't seem like a solution to just not have the check for blacklisted mails anymore...

Reply:
I think its sufficient since you cant create an Account with a blacklisted Email. So the check is not needed in my opinion.